src/extension/config.ts

```diff
@@ -70,38 +71,37 @@ export class Config {
         const showLineCoverage = rootConfig.get("showLineCoverage") as string;
         const showRulerCoverage = rootConfig.get("showRulerCoverage") as string;
 
-        interface IDefaultIcons {
-            gutterIconPathDark: string;
-            gutterIconPathLight: string;
-            noGutterIconPathDark: string;
-            noGutterIconPathLight: string;
-            partialGutterIconPathDark: string;
-            partialGutterIconPathLight: string;
+        const makeIcon = (colour: string): string | Uri => {
+            colour = colour
+                .trim()
+                .replace(/&/g, '&amp;')
+                .replace(/'/g, '&apos;')
+                .replace(/"/g, '&quot;')
+                .replace(/</g, '&lt;')
+                .replace(/>/g, '&gt;')
+                .replace(/\r\n/g, '&#13;')
+                .replace(/[\r\n]/g, '&#13;');
+            if (!colour) {
+                return "";
+            }
+
+            const svg = '<svg width="32" height="48" viewPort="0 0 32 48" xmlns="http://www.w3.org/2000/svg"><polygon points="16,0 32,0 32,48 16,48" fill="' + colour + '"/></svg>';
```

Comment:
I think the tests will also need to be adjusted to support this (or if we support the old method and the new, then we could add a new test to cover this).
https://github.com/ryanluker/vscode-coverage-gutters/actions/runs/7951747196/job/21742231483?pr=436#step:6:128